lib/events/pgevents/pgevents.go

```diff
@@ -77,6 +77,37 @@ const (
 	AzureADAuth AuthMode = "azure"
 )
 
+const (
+	schemaV1Table = `CREATE TABLE events (
+		event_time timestamptz NOT NULL,
+		event_id uuid NOT NULL,
+		event_type text NOT NULL,
+		session_id uuid NOT NULL,
+		event_data json NOT NULL,
+		creation_time timestamptz NOT NULL DEFAULT now(),
+		CONSTRAINT events_pkey PRIMARY KEY (event_time, event_id)
+	);`
+	schemaV1CreationTimeIndexPg   = `CREATE INDEX events_creation_time_idx ON events USING brin (creation_time);`
+	schemaV1CreationTimeIndexCRDB = `CREATE INDEX events_creation_time_idx ON events (creation_time);`
+	schemaV1SessionIDIndex        = `CREATE INDEX events_search_session_events_idx ON events (session_id, event_time, event_id)
+		WHERE session_id != '00000000-0000-0000-0000-000000000000';`
+)
+
+func buildSchemas(isCockroach bool) []string {
+	sb := strings.Builder{}
```

Comment:
Can we just sum two consts? There's no requirement to keep the brin index in the middle, so we could just

```go
const (
  schemaV1 = `CREATE TABLE events (...);
CREATE INDEX events_search_session_events_idx ...;`
  schemaV1Postgres = schemaV1 + `
CREATE INDEX events_creation_time_idx ...;`
  schemaV1Crdb = schemaV1 + `
CREATE INDEX events_creation_time_idx ...;`
)